modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/index/RocksDbHashIndexStorage.java

```diff
@@ -369,4 +373,26 @@ public void finishCleanup() {
             busyLock.unblock();
         }
     }
+
+    @Override
+    public @Nullable RowId getLastBuildRowId() {
+        return busy(() -> {
+            throwExceptionIfStorageInProgressOfRebalance(state.get(), this::createStorageInfo);
```

Comment:
Same comment, it's much more convenient to read local field, rather than the entry from the storage

Reply:
I do not think that the operation of building an index will often occur, and does it make sense to keep an empty value in memory? If you insist, I can do caching.

Reply:
Try to fix it.